.github/workflows/release-production.yml

```diff
@@ -2,7 +2,7 @@ name: Release Production Environment
 
 on:
   push:
-    tags: [ v* ]
+    tags: [v*]
```

Comment:
I like the idea of formatting the files to a certain agreed upon format or something commonly accepted in the industry.  Do we have a code formatting standard for our projects?  If so I have not seen it.

Reply:
Hey @ebadiere thanks for the input. So the I put a set of predefined formatting rules to the `.prettierrc` file, for now the rules are 
```
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "semi": false,
  "singleQuote": true
}
```
But everybody can always modify these rules or add more if it's beneficial.